paper-input-container.html

```diff
@@ -417,6 +417,10 @@
       return Polymer.dom(this).querySelector(this._inputSelector);
     },
 
+    get __inputElementValue() {
```

Comment:
Ughhhhhhh i'll make it `protected`. I'm clearly super confused about behaviours vs elements vs private things. I'll get there :rabbit: